pandas/_libs/skiplist.pyx

```diff
@@ -25,10 +25,10 @@ np.import_array()
 # TODO: optimize this, make less messy
 
 cdef class Node:
-    cdef public:
-        double_t value
-        list next
-        list width
+    # cdef public:
+    #    double_t value
```

Comment:
why are you commenting these out? just to show what the structure is? I would remove this

Reply:
aren't you supposed to declare the member variables in the .pyx, and not the .pxd?

Reply:
> aren't you supposed to declare the member variables in the .pyx, and not the .pxd?

AFAICT the rule is that if the class is declared in the .pxd, the member variables need to be declared there (and only there)

> why are you commenting these out? just to show what the structure is? I would remove this
 
I leave these in place so we don't have to go back and check the pxd